merino-integration-tests/src/utils/test_tools.rs

```diff
@@ -235,7 +235,13 @@ impl TestReqwestClient {
 async fn setup_remote_settings_bucket(server: &str) -> Result<(String, String)> {
     let reqwest_client = reqwest::Client::new();
     let bucket_info: serde_json::Value = reqwest_client
-        .post(format!("{}/v1/buckets/", server))
+        // Unfortunately `/dev/kinto.ini` does not support `/buckets/*`
+        // in `kinto.changes.resources`, so we should use the same bucket
+        // (i.e. `/buckets/main`) to create our collections in and make sure
+        // remote settings behave as expected. We use PUT here to overwrite
+        // the bucket transparently each time, since we don't really care
+        // about the stored metadata.
```

Comment:
Will calling `PUT` here delete the underlying collections? This method gets called as a part of parallel test runs, so we don't want to delete the collections used by other tests. The integration tests are one of the primary users of `/dev/kinto.ini`, so we can change that to support the old method, if that's possible.

Reply:
Good point. I restored the POST

Reply:
> Will calling PUT here delete the underlying collections?

No it won't